pkg/ebpf/dns_stats.go

```diff
@@ -2,15 +2,18 @@ package ebpf
 
 import (
 	"sync"
+	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 )
 
 type dnsStats struct {
-	lastTransactionID uint16
 	// More stats like latency, error, etc. will be added here later
 	successfulResponses uint32
 	failedResponses     uint32
+	successLatency      uint64 // Stored in µs
+	failureLatency      uint64
```

Comment:
nit: can we call these `successLatencySum`  and `failureLatencySum` . Same comment for the payload.  This lines up with how we represent RTT in the backend.